.github/chatmodes/PAW-01A Spec Agent.chatmode.md

```diff
@@ -100,6 +137,120 @@ As the spec evolves and becomes clearer, refine the Work Title if needed:
 - Update WorkflowContext.md if the title changes
 - Inform the user when the Work Title is updated
 
+### Feature Slug Normalization
```

Comment:
This is a lot of text for a small part of the agent's duties. Please try to consolidate it to reduce context bloat.